DirectProgramming/C++SYCL_FPGA/ReferenceDesigns/svd/src/svd_demo.cpp

```diff
@@ -0,0 +1,139 @@
+#include <sycl/ext/intel/fpga_extensions.hpp>
+#include <sycl/sycl.hpp>
+#include <vector>
+
+#include "exception_handler.hpp"
+#include "svd.hpp"
+#include "svd_testcase.hpp"
+
+// clang-format off
+SVDTestcase<float, 4, 4> small_4x4(
```

Comment:
Why not generating these matrices on the fly?
If not possible, hide them inside an include